ansible/library/port_alias.py

```diff
@@ -78,20 +78,22 @@ def get_platform_type(self):
                     return value
         return None
 
-    def get_portconfig_path(self, asic_id=None):
+    def get_portconfig_path(self, slotid=None, asic_id=None):
         platform = self.get_platform_type()
         if platform is None:
             return None
-        if asic_id is None:
+        if asic_id is None or asic_id == '':
             portconfig = os.path.join(FILE_PATH, platform, self.hwsku, PORTMAP_FILE)
-        else:
+        elif slotid is None or slotid == '':
             portconfig = os.path.join(FILE_PATH, platform, self.hwsku, str(asic_id), PORTMAP_FILE)
+        else:
+            portconfig = os.path.join(FILE_PATH, platform, self.hwsku, str(slotid), str(asic_id), PORTMAP_FILE)
```

Comment:
Is the port_config.ini file different for a card with the same hwsku being in a different slot?